sdcm/nemesis.py

```diff
@@ -651,6 +672,49 @@ def _init_new_node_with_latency_calculation(self, old_node_ip, rack=0):
     def run_nodetool_repair_on_new_node(self, node):
         self.repair_nodetool_repair(node)
 
+    def disrupt_terminate_and_recover_node_kubernetes(self):  # pylint: disable=invalid-name
```

Comment:
Let's printout to the log (info)
Affected:
```
GCE instance -
K8s node        - 
Scylla Pod       X  <-
Scylla node     X
```

Reply:
go over all possible "terminate" methods in for loop and printout

Reply:
done